src/profiles.sh

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
 
-gnomeVersion="$(expr "$(gnome-terminal --version)" : '.* \(.*[.].*[.].*\)$')"
+#Fix by wangshuwei at 2017.12.27 PM 11:25
```

Comment:
Is it possible to remove this type of comment? Git is made to handle the history, no need to add authors in comments